src/frontend/src/icp/components/transactions/IcTransactionsCkEthereumListeners.svelte

```diff
@@ -5,7 +5,8 @@
 	import { ckEthereumNativeToken } from '$icp-eth/derived/cketh.derived';
 	import type { CanisterIdText } from '$lib/types/canister';
 	import type { OptionIcCkToken } from '$icp/types/ic';
-	import { token } from '$lib/derived/token.derived';
+
```

Comment:
ditto

Reply:
Webstorm seems to have introduced those...Thanks!